Packs/AzureStorageContainer/Integrations/AzureStorageContainer/blob.txt

```diff
@@ -0,0 +1 @@
+xsoar
```

Comment:
what's this?

Reply:
I have no idea, most likely pre-commit again removing it